src/test/ui/fmt/ifmt-bad-arg.stderr

```diff
@@ -5,10 +5,10 @@ LL |     format!("{}");
    |              ^^
 
 error: invalid reference to positional argument 1 (there is 1 argument)
```

Comment:
Does this need a `bless`ing or there are some cases where the message hasn't changed?

Reply:
This is about `format!("{1}", 1)`. It complains about a reference to `1` when there's only one argument. That remains unchanged.